primitives/src/vid/advz.rs

```diff
@@ -194,22 +198,17 @@ where
     type Commit = Output<H>;
     type Share = Share<P, V>;
     type Common = Common<P, V>;
+    type Payload = Payload;
 
-    fn commit_only<I>(&self, payload: I) -> VidResult<Self::Commit>
-    where
-        I: IntoIterator,
-        I::Item: Borrow<u8>,
-    {
+    fn commit_only(&self, payload: &Self::Payload) -> VidResult<Self::Commit> {
+        // Can't use `Self::poly_commits_hash()`` here because `P::commit()`` returns
+        // `Result<P::Commitment,_>`` instead of `P::Commitment`.
+        // There's probably an idiomatic way to do this using eg.
+        // itertools::process_results() but the code is unreadable.
         let mut hasher = H::new();
         let elems_iter = bytes_to_field::<_, P::Evaluation>(payload);
         for coeffs_iter in elems_iter.chunks(self.payload_chunk_size).into_iter() {
```

Comment:
`coeffs_iter` -> `evals_iter`? (the elements are no longer coefficients.)

Reply:
changed `coeffs_iter` -> `evals_iter` because it's (an iterator over) the evaluations of a polynomial.